plenum/common/stacks.py

```diff
@@ -90,7 +90,6 @@ def handle_listener_events(self):
     def restart(self):
         logger.warning("Stopping client stack on node {}".format(self))
         self.stop()
-        time.sleep(0.2)
```

Comment:
Don't we need this timeout anymore?

Reply:
This time out has gone to binding cycle if it is needed (i.e. busy port).

Reply:
But I was thinking that this timeout was added with a different purpose, to have a chance for sockets to be closed, isn't it?

Reply:
No, connection sockets are not closed during TIME_WAIT (~2 mins). This time out was added just for sure that the kernel freed listener socket structure. But our tests under syn flood shows that it is not enough. So I've implemented a logic to have more chances to re-bind in case of immediate stack restart.